### PR TITLE
[stable/mysql] deployment annotations

### LINKED ISSUE #15052: mysql deployment annotations
when using helm umbrela chart and using mysql chart,
helm hooks cannot be assign in deployment metadata resource.
need to template annotations sections.

### PR DESCRIPTION
**Pull request by Amos-85**
#### What this PR does / why we need it:
It will give us the ability to use helm hooks.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #15052

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/mysql/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mysql
-version: 1.2.1
+version: 1.2.2
```
Comment (olemarkus) [inline stable/mysql/Chart.yaml]: Can you bump the minor version here as a new feature has been added

**Review by olemarkus — CHANGES_REQUESTED** [review]

File: stable/mysql/Chart.yaml (hunk 1 above)
Comment (Amos-85) [reply stable/mysql/Chart.yaml]: @olemarkus , I have bump to version 1.2.3

Comment (Amos-85) [reply stable/mysql/Chart.yaml]: @olemarkus ,
may you please review?

Comment (olemarkus) [reply stable/mysql/Chart.yaml]: Hey. The minor version still needs to be bumped

Comment (Amos-85) [reply stable/mysql/Chart.yaml]: Thanks @olemarkus ,
Version bumped to 1.3.0

**Pull request merged**